changelog.md

```diff
@@ -54,7 +54,12 @@
 * [2256](https://github.com/zeta-chain/node/pull/2256) - fix rate limiter falsely included reverted non-withdraw cctxs
 
 ### CI
-* [2268](https://github.com/zeta-chain/node/pull/2268) - CI: updated the publish-release pipeline to utilize the Github Actions Ubuntu 20.04 Runners.
+
+* [2285](https://github.com/zeta-chain/node/pull/2285) - added nightly EVM performance testing pipeline.
+* [2285](https://github.com/zeta-chain/node/pull/2285) - modified localnet testing docker image to utilitze debian:bookworm.
+* [2285](https://github.com/zeta-chain/node/pull/2285) - removed build-jet runners where applicable.
+* [2285](https://github.com/zeta-chain/node/pull/2285) - removed deprecated/removed upgrade path testing pipeline.
```

Comment:
There should be only one entry per pr, maybe condense the task list into a sentence?